src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -24,24 +24,26 @@
         "watch": "tsc -p . --watch",
         "pack": "npx webpack",
         "lint-cadl-library": "cadl compile . --warn-as-error --import @cadl-lang/library-linter --no-emit",
-        "test": "mocha",
+        "test": "mocha dist/test/**/*.js",
         "test-official": "c8 mocha --forbid-only",
         "lint": "eslint . --ext .ts --max-warnings=0",
         "lint:fix": "eslint . --fix --ext .ts",
-        "prettier:fix": "prettier --write --config .prettierrc.json ./src/**/*.ts",
+        "prettier:fix": "prettier --write --config .prettierrc.json ./src/**/*.ts ./test/**/*.ts",
```

Comment:
Put this to "prettier" as well.

Reply:
Thanks @pshao25  Good suggestion. I will add.